docker-storage-setup.sh

```diff
@@ -58,7 +58,7 @@ POOL_LV_NAME="docker-pool"
 DOCKER_ROOT_LV_NAME="docker-root-lv"
 DOCKER_ROOT_DIR="/var/lib/docker"
 
-DOCKER_STORAGE="/etc/sysconfig/docker-storage"
+STORAGE="/etc/sysconfig/docker-storage"
```

Comment:
Out about calling this variable `STORAGE_OUT_FILE`. This will fit nicely with your follow up patches as well. Just using `STORAGE` is too generic.